be/src/controlers/rest.router.ts

```diff
@@ -64,7 +82,11 @@ router.get('/vocab/:id/entities', (req: Request, res: Response) => {
 });
 
 router.post('/vocab/:id/entities', (req: Request, res: Response) => {
-  res.json({statusCode:206});
+    const vocabID: string = req.params.id;
+    const entity: Entity = createNewEntityDBO(vocabID, req.body as EntityDTO);
+    entityService.createEntity(vocabID, entity)
+        .then(entity => res.json(entity))
```

Comment:
Status-Code should be `201 Created`
```suggestion
        .then(entity => res.status(201).json(entity))
```